langkit/support/langkit_support-token_data_handlers.ads

```diff
@@ -285,4 +294,31 @@ package Langkit_Support.Token_Data_Handlers is
      (TDH : Token_Data_Handler; Line_Number : Positive) return Text_Type;
    --  Get the source text of line at index ``Line_Number``
 
+   function Get_Sloc
+     (TDH : Token_Data_Handler; Offset : Natural) return Source_Location;
+   --  Return the sloc for given ``Offset`` in ``TDH``. If:
```

Comment:
This is not an offset, but an index of a codepoint in TDH.Source_Buffer, so I think we should rename it accordingly